smartlearner/views.py

```diff
@@ -51,3 +51,39 @@ def mean(self):
     @property
     def stderror(self):
         return ItemGetter(self, attribute=1)
+
+
+class ReconstructionError(View):
+    def __init__(self, predict_fct, dataset):
+        super().__init__()
+
+        batch_size = 1024  # Internal buffer
```

Comment:
It would be better if `batch_size` was an optional parameter instead of an hard coded value. @ASalvail while at it we should also change the one in `ClassificationError`. This would fix #5